src/webots/scene_tree/WbFieldDoubleSpinBox.cpp

```diff
@@ -66,12 +66,12 @@ void WbFieldDoubleSpinBox::stepBy(int steps) {
       value += pow(10, -mDecimals) * steps;
       break;
     case RADIANS: {
-      // angle in radians: change by pi/24 and snap to pi/24 if close by 1%
+      // angle in radians: change by pi/24 and snap to the nearest multiple of PI
       value += M_PI * steps / 24.0;
       double intpart;
       double decpart = modf(fabs(value) / M_PI, &intpart);
```

Comment:
Would it be possible to get rid of intpart, as it seems not used anymore?

Reply:
indeed we can!